### DISCUSSION TITLE
FCOS update??

### DISCUSSION THREAD

**Discussion by wiqu85** [okd-project/okd#2024] [Q&A] [answered]
Hi! Will there be further okd updates on fcos? The current stable version is from 03/2024 and has many vulnerabilities

**Comment by Tyrone-W** [comment, marked as answer]
There will be no further FCOS OKD builds than the one you mention, please take a look at https://okd.io/blog, specifically https://okd.io/blog/2024/06/01/okd-future-statement/ for the current status. This is discussed in the okd working group which takes place every other Tuesday at 12:00PM Eastern-US, info on joining etc can be found at https://okd.io/docs/community/working-groups
